manager/src/main/java/com/redhat/developer/manager/ProcessorService.java

```diff
@@ -23,13 +27,9 @@
     @Inject
     BridgesService bridgesService;
 
-    @Transactional
     public Processor createProcessor(String bridgeId, String customerId, ProcessorRequest processorRequest) {
         Bridge bridge = bridgesService.getBridge(bridgeId, customerId);
-        if (BridgeStatus.AVAILABLE != bridge.getStatus()) {
-            /* We cannot deploy Processors to a Bridge that is not Available */
-            throw new BridgeLifecycleException("Bridge with id '" + bridge.getId() + "' for customer '" + customerId + "' is not in the '" + BridgeStatus.AVAILABLE + "' state.");
-        }
+        checkBridgeInActiveStatus(bridge);
```

Comment:
What about move this logic inside `bridgesService`? We can probably add a new method like `getActiveBridge` or similar 🤔 
Wdyt?

Reply:
I'm OK with it in the `ProcessorService` for now. There isn't currently anywhere else we'd need to use this logic at this point. If that changes, we can move it 👍